compiler/src/iree/compiler/Codegen/Common/TileAndDistributeToWorkgroupsPass.cpp

```diff
@@ -302,24 +306,42 @@ struct LowerDispatchWorkgroupCountFromSetEncodingOp
 struct TileAndDistributeToWorkgroupsPass
     : public TileAndDistributeToWorkgroupsBase<
           TileAndDistributeToWorkgroupsPass> {
+  TileAndDistributeToWorkgroupsPass(int32_t maxWorkgroupParallelDims)
+      : maxWorkgroupParallelDims(maxWorkgroupParallelDims) {}
   void getDependentDialects(DialectRegistry &registry) const override {
     registry
         .insert<AffineDialect, IREE::Flow::FlowDialect, IREE::HAL::HALDialect,
                 linalg::LinalgDialect, IREE::LinalgExt::IREELinalgExtDialect,
                 scf::SCFDialect, tensor::TensorDialect>();
   }
 
+  void initOptions() {
```

Comment:
This is non-standard way of doing this, either do what is done here

https://github.com/openxla/iree/blob/663f92cd71ed3a85ca677e847d2446ff48423080/compiler/src/iree/compiler/Dialect/Flow/Transforms/FormDispatchRegions.cpp#L782

(there are other ways too, but this is what I can point to immediately.